app/controllers/requested_accounts_controller.rb

```diff
@@ -42,8 +42,19 @@ def enable_account_requests
     @course.save
   end
 
+  # List of requested accounts for a user's courses.
+  def index
+    if user_signed_in?
+      @courses = current_user.courses.select do |course|
```

Comment:
I think this would be more useful as an unconditional list of requested accounts, rather than limited to the viewing admin's courses. In most cases, the desired course of action is just to create all of the requested accounts... the only time we wouldn't is if there was someone trying to create a bunch of accounts with bad names, or someone was mass-requesting so many that it's obviously not legit. Minimizing the time until the request is processed  and avoiding cases where requests sit around without being handled are the main goals.

Reply:
In addition to the above buttons then, should we also have a "Create all accounts" button?

Reply:
I think that would be good.

Reply:
Will there be an issue here with using the backup account at all? I imagine this button would often get us over the 6 accounts limit. Last I remember we had a limited amount of additional accounts we could create with our backup account.

Reply:
The backup account isn't set on Wiki Education Dashboard, but it should be used if necessary on Programs & Events Dashboard.

I believe that the Wiki Education Dashboard is not subject to those IP limits anyway, because its IP is whitelisted.